### ISSUE TITLE
How to get serial number of hard-disk in `GVCL`

### ISSUE THREAD

**Issue by zanderwong** [suiyunonghen/GVCL#1]
Would you like to tell me if `GVCL` can get the serial number of hard-disks?  
like this:
disk1: S2TTNWAGA11265K
disk2: E203421K3AUV5S
...

**Comment by suiyunonghen** [comment]
No,GVCL is only UI Controls，and serial number of hard-disks is not GVCl Work
maybe sometime i will Add this support,but not now
